crates/bevy_core/src/bytes.rs

```diff
@@ -46,7 +42,7 @@ pub trait FromBytes {
 
 impl<T> FromBytes for T
 where
-    T: Byteable + Copy,
+    T: Byteable,
```

Comment:
Nice catch on not needing the `Copy`/`Sized` requirements since `Byteable` already requires `Copy`/`Sized`! 😄